pkg/runner/flags.go

```diff
@@ -325,6 +326,7 @@ func checkForRemovedConfigFlags(fs *flag.FlagSet, args []string) error {
 func handleOldFlags(oldFlag, newFlag string) {
 	if oldFlag != "migrate" {
 		log.Warn("msg", fmt.Sprintf("Update removed flag `%s` to new flag `%s`", oldFlag, newFlag))
+		return
```

Comment:
What you are saying is donde by the method `checkForRemovedConfigFlags` which goes through all the old flags, and for each old flag, calls this `handleOldFlags` method that just logs the `Warn` message. 

If we don't return here, then it will go over the `migrate flag` switch case. Since we are in this part of the branch because `oldFlag != "migrate"` it will always print the `default` case.